### ISSUE TITLE
WOOL Editor - Bug: Switching to the editor from another application

### ISSUE THREAD

**Issue by dennishofs** [woolplatform/wool#90]
**Editor Version:**
v1.10.2

**Desktop or Web version:** 
Desktop

**Operating System (Desktop) or Browser (Web):**
Windows

**Describe the Issue:**
When switching from another application to the WOOL editor with Alt-Tab or clicking in the task bar, I always have to do it twice. The first time the editor icon lights up in the task bar and only the second time the window comes to the foreground.

**Comment by borisvanschooten** [comment]
Fixed in next release.  This is a problem with the focus workaround I implemented.  It triggers another windowing bug in Electron.  I work around it now by minimizing instead of blurring the window before re-focusing it. Side effect is that the window shows an unminimize animation.

https://github.com/electron/electron/issues/19977 - focus problem that it worked around
https://github.com/electron/electron/issues/2867 - the other bug, now also worked around
